src/components/movie-view/movie-view.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export class MovieView extends React.Component {
+	constructor() {
+		super();
+
+		this.state = {};
+	}
+
+	render() {
+		const { movie, onClick, button } = this.props;
```

Comment:
remove destructing onClick, button. it never used

Reply:
I forgot I added these when working on the back button. They've been removed. Thank you!